cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1019,7 +1029,9 @@ public MapAndStatus(T loadMap, K status) {
             return requestRebalance(reconciliation, host, apiClient, kafkaRebalance, true, rebalanceOptionsBuilder);
         } else {
             LOGGER.warnCr(reconciliation, "{}: Ignore annotation {}={}", ANNO_STRIMZI_IO_REBALANCE, rebalanceAnnotation);
-            return configMapOperator.getAsync(kafkaRebalance.getMetadata().getNamespace(), kafkaRebalance.getMetadata().getName()).compose(loadmap -> Future.succeededFuture(new MapAndStatus<>(loadmap, buildRebalanceStatusFromPreviousStatus(kafkaRebalance.getStatus(), validate(reconciliation, kafkaRebalance)))));
+            Set<Condition> conditions = validate(reconciliation, kafkaRebalance);
+            conditions.add(StatusUtils.buildWarningCondition("InvalidAnnotation", "Wrong annotation value: " + rebalanceAnnotation + ". The rebalance resource supports the following annotations in `Ready` state: refresh"));
```

Comment:
Same as above, how do you know any annotation is set?

Reply:
Resolved this issue in new changes